app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -1,9 +1,10 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
 
 export default class ApartmentIndex extends Component {
   render() {
     return (
       <h3>ApartmentIndex</h3>
     )
+    return <h3>Apartment Listings</h3>;
   }
 }
```

Comment:
I suspect that this will break your app. Double check and let me know if it works.

Reply:
I removed line 8, return apartment listing

Reply:
added .DS store to .gitignore. Couldn't find the .DS Store files in the app, but found on terminal tried to delete.